frontend/src/components/Profile.vue

```diff
@@ -0,0 +1,184 @@
+
+            <div class="container__city">
+              <select name="City" id="city" required>
+                <option value="City" disabled selected>City*</option>
+                <option value="Brussels">Brussels</option>
+                <option value="Leuven">Leuven</option>
+                <option value="Namur">Namur</option>
+                <option value="Antwerp">Antwerp</option>
+                <option value="Liège">Liège</option>
+                <option value="Ghent">Ghent</option>
+                <option value="Brugge">Brugge</option>
+                <option value="Eupen">Eupen</option>
+                <option value="Arlon">Arlon</option>
+              </select>
+              <span class="container__select-icon"
+                ><i class="zmdi zmdi-chevron-down"></i
+              ></span>
+            </div>
+          </div>
+        </div>
+
+        <div class="container__radio">
+          <label for="usertype" class="radio-label">Status*</label>
+          <div class="container__radio-item1">
+            <input type="radio" name="status" id="newcomer" checked />
+            <label for="newcomer">Newcomer</label>
+            <span class="container__radio-check"></span>
+          </div>
+
+          <div class="container__radio-item1">
+            <input type="radio" name="status" id="volunteer" />
+            <label for="volunteer">Volunteer</label>
+            <span class="container__radio-check"></span>
+          </div>
+        </div>
+
+        <div class="container__group">
+          <input
+            type="email"
+            placeholder="E-mail*"
+            name="email"
+            id="email"
+            required
+          />
+        </div>
+        <div class="container__group">
+          <input
+            type="password"
+            placeholder="Password*"
+            name="password"
+            id="password"
+            required
+          />
+        </div>
+
+        <div class="container__group">
+          <div class="container__select">
+            <select name="Gender" id="gender" required>
+              <option value="Gender" disabled selected>Gender*</option>
+              <option value="Male">Male</option>
+              <option value="Female">Female</option>
+              <option value="Other">Other</option>
+            </select>
+            <span class="container__select-icon"
+              ><i class="zmdi zmdi-chevron-down"></i
+            ></span>
+          </div>
+        </div>
+
+        <div class="container__group">
+          <input
+            type="text"
+            placeholder="Nationality"
+            name="nationality"
+            id="nationality"
+          />
+        </div>
+
+        <div class="container__group">
+          <div class="container__select">
+            <select name="languages" id="languages">
+              <option value="Language" disabled selected>Languages</option>
+              <option value="French">English</option>
```

Comment:
Same here about languages, like with gender

Reply:
replace with textbox?

Reply:
I would say yes, and in the text box there will be current information. on clicking EDIT button, these text boxes should be changed to SELECT